### PR TITLE
CSS validator/linter t9 GitHub Actions workflow

### PR DESCRIPTION
**Pull request by Programmer80101**
## CSS Linter
Add CSS linter to GitHub Actions workflow. It automatically catches CSS typos, errors, and issue.

It identified a **significant** number of *issues*.
Fix issue #80.
You can fix some of those errors by running the following code:
```shell
npx stylelint **/*.css --fix
```

### CODE REVIEW

File: .github/workflows/stylelint.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install stylelint
+        run: npm install -g stylelint stylelint-config-standard
+
+      - name: Run stylelint
+        run: stylelint "**/*.css"
```
Comment (postmodern) [inline .github/workflows/stylelint.yml:22]: Remove the double quotes. They will treat the expression `**/*.css` as literal. Unless the `stylelint` command interprets a glob expression?

**Review by postmodern — CHANGES_REQUESTED** [review]
Noticed one minor thing with argument quoting/

File: .github/workflows/stylelint.yml (hunk 1 above)
Comment (Programmer80101) [reply .github/workflows/stylelint.yml]: You're right, I removed the double quotes!

**Comment by postmodern** [comment]
@Programmer80101 any reason why we shouldn't use the [actions-hub/stylelint](https://github.com/actions-hub/stylelint) GitHub Actions action?

**Comment by Programmer80101** [comment]
> @Programmer80101 any reason why we shouldn't use the [actions-hub/stylelint](https://github.com/actions-hub/stylelint) GitHub Actions action? This looks ready to be merged, but just wanted to make sure there was a reason to not use the stylelint GitHub action.

Not using stylelint Github action allows more customization and flexibility. Using stylelint GitHub action is definitely better in some ways.

**Pull request merged**